src/controllers/secure.register.filter.controller.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response, NextFunction } from "express";
+import * as config from "../config";
+import { logger } from "../utils/logger";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.SECURE_REGISTER_FILTER_PAGE}`);
+    return res.render(config.SECURE_REGISTER_FILTER_PAGE, {
+      backLinkUrl: config.SOLD_LAND_FILTER_URL
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  logger.debugRequest(req, `POST ${config.SECURE_REGISTER_FILTER_PAGE}`);
+  try {
+    if (req.body.secure_register === '1') {
```

Comment:
Similar to what Moh said on my PR you could put this into it's own variable to use in the if statements